src/execute_linter.h

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef SRC_EXECUTE_LINTER_H_
+#define SRC_EXECUTE_LINTER_H_
+
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "absl/strings/string_view.h"
+#include "src/lint_errors.h"
+#include "src/linter.h"
+#include "src/linter_options.h"
+
+namespace zetasql::linter {
+
+// This function will parse "NOLINT"(<CheckName>) syntax from
```

Comment:
We only need function comment in header files. Please consider removing the duplicate comments from cc files.

Reply:
Sure.